SB2018/2018-01-12.md

```diff
@@ -1,12 +1,12 @@
 ---
-to: All teams
+to: All teams but Southend and Westcliff
```

Comment:
Are we going to invite these teams to Tech Days, or are we assuming that they're too far away?

IMO unless they've already said so, we should invite them and let them decide whether it's too far

Reply:
Agreed. All teams should be on announcement emails like this. It's up to them to decide if it's relevant or not.

Reply:
Apologies for the confusion here, The issue is we can't really mentor those teams. I was thinking we'd remove the mentoring section of the email for them, or at least rephrase it. 

de2e94a rephrases it in a separate email, but I don't like this because there's now duplication of words

Reply:
> I was thinking we'd remove the mentoring section of the email for them, or at least rephrase it.

I think that's fine, though I'd be in favour of including the details of that in this repo. I see this repo as being most useful when it contains exactly what we actually send out (modulo actual email addresses; maybe sign-off footers), rather than merely approximations.

I think it would also be fine to include meta-instructions inline, maybe with something like jinja templating syntax (just as a standard to use; I'm not necessarily suggesting that we actually use jinja):
``` jinja
{% if team not in ('southend', 'westcliff') %}
... mentoring ...
{%  else %}
... unfortunately remote mentoring only ..
{% endif %}
```

Whatever we do, being clear about this sort of thing upfront will help reduce the overhead on everyone and enable us to get through review faster.